test/fixtures/poro.rb

```diff
@@ -1,7 +1,9 @@
 class Model < ActiveModelSerializers::Model
-  rand(2).zero? && derive_attributes_from_names_and_fix_accessors
-
-  attr_writer :id
+  if rand(2).zero?
+    derive_attributes_from_names_and_fix_accessors
+  else
+    attr_writer :id
```

Comment:
I found when using `ActiveModelSerializers::Model.derive_attributes_from_names_and_fix_accessors ` and `class SomeName < ActiveModelSerializers::Model; attributes :something; end` in my own code, that `SomeName.new(id: 'uuid')` failed, as there was no attr_writer